pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -283,7 +292,7 @@ in (buildEnv {
   # Get rid of all log files. They are not needed, but take up space
   # and render the build unreproducible by their embedded timestamps.
   ''
-    find $TEXMFSYSVAR/web2c -name '*.log' -delete
+    find "$TEXMFSYSVAR"/web2c -name '*.log' -delete
```

Comment:
Not relevant for this PR

Reply:
Yeah, that was just linting on my part, I can take it out.

Reply:
I'll keep the lint – we should always quote strings!